integration-cli/docker_cli_logs_test.go

```diff
@@ -180,45 +126,26 @@ func (s *DockerSuite) TestLogsStderrInStdout(c *check.C) {
 
 func (s *DockerSuite) TestLogsTail(c *check.C) {
 	testLen := 100
-	runCmd := exec.Command(dockerBinary, "run", "-d", "busybox", "sh", "-c", fmt.Sprintf("for i in $(seq 1 %d); do echo =; done;", testLen))
-
-	out, _, _, err := runCommandWithStdoutStderr(runCmd)
-	if err != nil {
-		c.Fatalf("run failed with errors: %s, %v", out, err)
-	}
+	out, _ := dockerCmd(c, "run", "-d", "busybox", "sh", "-c", fmt.Sprintf("for i in $(seq 1 %d); do echo =; done;", testLen))
 
 	cleanedContainerID := strings.TrimSpace(out)
-	exec.Command(dockerBinary, "wait", cleanedContainerID).Run()
+	dockerCmd(c, "wait", cleanedContainerID)
 
-	logsCmd := exec.Command(dockerBinary, "logs", "--tail", "5", cleanedContainerID)
-	out, _, _, err = runCommandWithStdoutStderr(logsCmd)
-	if err != nil {
-		c.Fatalf("failed to log container: %s, %v", out, err)
-	}
+	out, _ = dockerCmd(c, "logs", "--tail", "5", cleanedContainerID)
 
 	lines := strings.Split(out, "\n")
 
 	if len(lines) != 6 {
 		c.Fatalf("Expected log %d lines, received %d\n", 6, len(lines))
 	}
-
-	logsCmd = exec.Command(dockerBinary, "logs", "--tail", "all", cleanedContainerID)
-	out, _, _, err = runCommandWithStdoutStderr(logsCmd)
-	if err != nil {
-		c.Fatalf("failed to log container: %s, %v", out, err)
-	}
+	out, _ = dockerCmd(c, "logs", "--tail", "all", cleanedContainerID)
 
 	lines = strings.Split(out, "\n")
 
 	if len(lines) != testLen+1 {
 		c.Fatalf("Expected log %d lines, received %d\n", testLen+1, len(lines))
 	}
-
-	logsCmd = exec.Command(dockerBinary, "logs", "--tail", "random", cleanedContainerID)
-	out, _, _, err = runCommandWithStdoutStderr(logsCmd)
-	if err != nil {
-		c.Fatalf("failed to log container: %s, %v", out, err)
-	}
+	out, _, _ = dockerCmdWithStdoutStderr(c, "logs", "--tail", "random", cleanedContainerID)
```

Comment:
But if we use `dockerCmd`, the `out`  is combined output of `stdout` and `stderr`, and I think we just need `stdout` here